docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,76 @@
+  * Version check
+  * User-Agent Tool
+
+## Summary Data
+
+In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month).
+
+For For every namespace, repo, and reference (tag or digest):
+
+  * Unique IP address count
+  * Pulls by tag count
+  * Pulls by digest count
+  * Version Check count
+
+## Data Definitions
+
+| Data Point | Definition |
+|:-----|:--------|
+| Action | An Action represents the multiple request events associated with a docker pull. We have applied rules to these events so that the data is more meaningful in analyzing user behavior and intent. An Action can be filtered into three distinct categories: version check, pull by tag, and pull by digest. Each Action is represented as a single row in the raw data. |
+| Version Check  |  This is a filter on the Action data point. It is a speculation of user intent.                                                    * Includes: HEAD by tag not followed by a GET (by the same IP address within 5-second window) * Excludes: HEAD by digest|
+| Pull by Tag | This is a filter on the Action data point. It is a speculation of user intent. Includes: GET (by digest or by tag)If the GET is immediately preceded by a HEAD by tag (by the same IP address within a 5-second window), then the GET and HEAD together are counted as 1 pull by tag. If the GET by tag is immediately followed by another GET (by the same IP address within a 5-second window, but a different digest), then both GETs together are counted as 1 pull by tag.|
+|Pull by Digest  |This is a filter on the Action data point. It is a speculation of user intent. Includes: GET by digest. If the GET is immediately preceded by a HEAD by digest (by the same IP address within a 5-second window), then the GET and HEAD together are counted as 1 pull by digest. If the GET is immediately followed by another GET (same IP address within a 5-second window, but a different digest), then both GETs together are counted as 1 pull by digest. Includes: HEAD by digest, not followed by a GET|
+|Type | The industry from which the event originates. Industry types include “business”, “ISP” (internet service provider), “hosting”, “education”, and “null” in cases where the industry could not be identified.|
+|Host |The cloud service provider used in an event.|
+|Country |The country from which the request originated.  |
+|Timestamp |Date & time of an event in the following schema: YYYY-MM-DD 00:00:00|
+|Namespace |The Docker organization that a repository is a part of. |
+|Repo |The repository that an image belongs to.|
+|Reference |The tag or digest of any given image.|
+|HTTP Method | The HTTP method(s) used in a request by the client. More information on Docker Registry HTTP API protocols can be found here.|
+|User-Agent Tool|The application a user used to pull an image.  Extracted from the UA string.|
```

Comment:
I added User Agent Version because I can see that in the CSV I export from the GUI. I can also see "Action Day", but I am not seeing the "Unique IP Addresses" counter in the CSV. Maybe we need to double check that these data points are correct?

```suggestion
|User Agent Tool|The application a user used to pull an image.  Extracted from the UA string.|
|User Agent Version|The application version, extracted from the UA string.|
```

Reply:
The columns vary a little between raw data and summary data. For instance "Unique IP Addresses" only makes sense in the summary data, since the raw data is made up of single actions originating from single IPs.

Probably worth adding `Action Day` in though. I'm not sure it's the best name, but it just refers to the date portion of the timestamp.